src/interface/gemm_interface.hpp

```diff
@@ -19,12 +19,10 @@
  *
  *  SYCL-BLAS: BLAS implementation using SYCL
  *
- *  @filename blas3_interface.hpp
- *
  **************************************************************************/
 
-#ifndef SYCL_BLAS_BLAS3_INTERFACE_HPP
-#define SYCL_BLAS_BLAS3_INTERFACE_HPP
+#ifndef SYCL_BLAS_BLAS3_GEMM_HPP
+#define SYCL_BLAS_BLAS3_GEMM_HPP
```

Comment:
```suggestion
#define SYCL_BLAS_GEMM_INTERFACE_HPP
```

To match the file name